release_notes/v2.3.1.txt

```diff
@@ -0,0 +1,22 @@
+v2.3.1 
+--------------------------
+
+Release Notes
+-------------
+This v2.3.1 Release is the LTS version of the fabric-chaincode-java
```

Comment:
v2.3.1 isn't LTS :-)  Just change this to say it's a bug fix release of the main branch.